src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -1328,6 +1329,10 @@ error_directive_trivia
   : '#' 'error'
   ;
 
+ignored_directive_trivia
+  : '#' ':'
```

Comment:
shouldn't there be content that follows this?

Reply:
this is same as shebang trivia

Reply:
i hate it.  but works for me :)